backoffice/src/components/sidebar/index.js

```diff
@@ -1,7 +1,9 @@
 import { MetaComponent } from '@rebelstack-io/metaflux';
 import { instanceElement } from '../../utils';
-import defaulAvatar from '../../assets/images/user.png';
-import cog from '../../assets/icons/cog-solid.svg';
+import defaulAvatar from '../../images/user.svg';
+import cog from '../../css/icons/cog-solid.svg';
```

Comment:
did you change the icons location with a new path or just update the paths to match your paths in you dev env?  i created a assets folder since we are using the  image loader (with webpack) otherwise we need to put the images path fixed in the public folder... but that means we need to update the styles to use the images in the public folder insstead of  use the dinamyc ` import imgSrc from ../...` .  If you changed the paths this change may broke something ...